NetCasbin/InternalEnforcer.cs

```diff
@@ -17,34 +17,29 @@ public class InternalEnforcer : CoreEnforcer
         /// <returns></returns>
         protected bool AddPolicy(string sec, string ptype, List<string> rule)
         {
-            var ruleAdded = model.AddPolicy(sec, ptype, rule);
-            if (!ruleAdded)
+            if (model.HasPolicy(sec, ptype, rule))
             {
-                return false;
-            } 
-
+                return true;
+            }
+            
```

Comment:
Remove spaces.

Reply:
I think these spaces are necessary, the method body look like this:
```csharp
protected bool AddPolicy(string sec, string ptype, List<string> rule)
{
    if (model.HasPolicy(sec, ptype, rule))
    {
        return false;
    }

    if (adapter != null && autoSave)
    {
        try
        {
            adapter.AddPolicy(sec, ptype, rule);
        }
        catch (NotImplementedException)
        {
            // error intentionally ignored
        }

        watcher?.Update();
    }

    var ruleAdded = model.AddPolicy(sec, ptype, rule);
    return ruleAdded;
}
```

Reply:
By spaces I mean the " " (space chars) in the empty line:

![image](https://user-images.githubusercontent.com/3787410/84393653-6cad6f00-ac2e-11ea-9cde-4406b4733ee7.png)

Reply:
Ok, I removed the spaces.